workspaces/scorecard/plugins/scorecard-backend-module-jira/config.d.ts

```diff
@@ -21,6 +21,10 @@ export interface Config {
     plugins?: {
       /** JIRA datasource configuration */
       jira?: {
+        options?: {
+          mandatoryFilter?: string;
+          customFilter?: string;
+        };
```

Comment:
Please specific all the attributes you read from the Config into this object. Also the `jira` attributes.

Since I've learned now that the `jira` configuration isn't used by any other plugin, I'm not sure if we should maybe move it to `scorecard.plugins.jira`

Wdyt @imykhno @dzemanov ?